.github/pull_request_template.md

```diff
@@ -85,3 +85,24 @@ If this is a community PR, the Airbyte engineer reviewing this PR is responsible
 - Documentation which references the generator is updated as needed
 
 </details>
+
+<details><summary><strong>Updating the Python CDK</strong></summary>
+
+### Airbyter
+
+Before merging:
+- [ ] Pull Request description explains what problem it is solving
+- [ ] Code change is unit tested
+- [ ] Build and my-py check pass
+- [ ] Smoke test the change on an affected connector
+   - On Github: Run [this workflow](https://github.com/airbytehq/airbyte/actions/workflows/connectors_tests.yml), passing `--use-local-cdk --name=source-<connector>` as options
+   - Locally: `airbyte-ci connectors --use-local-cdk --name=source-<connector> test`
+- [ ] PR is reviewed and approved
+      
+After merging:
+- [ ] [Publish the CDK](https://github.com/airbytehq/airbyte/actions/workflows/publish-cdk-command-manually.yml)
+   - The CDK does not follow proper semantic versioning. Choose minor if this the change has significant user impact or is a breaking change. Choose patch otherwise.
+   - Write a thoughtful changelog message so we know what was updated.
+- [ ] (Optionally) Merge the platform PR created for the Connector Builder
```

Comment:
This is auto-created, right? Small wording suggestion if so.

```suggestion
- [ ] (Optionally) Merge the platform PR that was auto-created for updating the Connector Builder's CDK version.
```

As an aside, is this something we should actually be encouraging people to do? Or are we still happy with people just bumping the CB's CDK version as needed?

Reply:
>  is this something we should actually be encouraging people to do? Or are we still happy with people just bumping the CB's CDK version as needed?
Good question. I re-worded so that it's optional if the change does not affect the connector builder, but I don't feel too strongly about it either way